src/test/kotlin/me/lensvol/blackconnect/BlackdClientTestCase.kt

```diff
@@ -35,7 +35,7 @@ class BlackdClientTestCase : BlackConnectTestCase() {
             )
         }
 
-        val blackdClient = BlackdClient("localhost", 45484)
+        val blackdClient = BlackdClient("localhost", 45484, false)
```

Comment:
We can [pass the default value in the constructor](http://kotlin-quick-reference.com/102a-R-constructors-default-values.html) or do it explicitly at the instantiation:

```BlackdClient("localhost, 45484, useSSL = false)```